compiler/one-cmds/one-quantize

```diff
@@ -360,6 +391,33 @@ def _quantize(args):
         # run circle-quantizer
         _utils._run(circle_quantizer_cmd, err_prefix="circle_quantizer", logfile=f)
 
+        # evaluate
+        if _utils._is_valid_attr(args, 'evaluate_result'):
+            circle_eval_diff_path = os.path.join(dir_path, 'circle-eval-diff')
+            quant_model = ""
+            if _utils._is_valid_attr(args, 'output_path'):
+                quant_model = getattr(args, 'output_path')
+            tmp_fake_quant_model = os.path.join(
+                tmpdir,
+                os.path.splitext(os.path.basename(args.input_path))[0]) + '.fake_quant.circle'
+
+            # do fake quantization
+            fake_quantize_cmd = Command(circle_quantizer_path, args, f)
+            fake_quantize_cmd.add_noarg_option_if_valid_arg('--verbose', 'verbose') \
+                .add_option_with_values('--fake_quantize', [quant_model, tmp_fake_quant_model]) \
+                .run()
+
+            # compare fake-quant model and fp32 model
+            circle_eval_diff_cmd = Command(circle_eval_diff_path, args, f)
+            circle_eval_diff_cmd.add_option_with_valid_args('--first_model', ['input_path']) \
+                .add_option_with_values('--second_model', [tmp_fake_quant_model]) \
+                .add_option_with_valid_args('--first_input_data', ['test_data']) \
+                .add_option_with_valid_args('--second_input_data', ['test_data']) \
+                .add_option_with_valid_args('--input_data_format', ['input_data_format']) \
+                .add_noarg_option_if_valid_arg('--print_mae', 'print_mae') \
+                .add_noarg_option_if_valid_arg('--print_mape', 'print_mape') \
+                .add_noarg_option_if_valid_arg('--print_mpeir', 'print_mpeir') \
+                .run()
```

Comment:
Hmmm.. I think I understand why you implement `Command` class and it's a very good idea. But, IMHO, I think the name of methods are a bit verbose and seems that they even don't need to be called to me because `args` already has the contents for the command.

Can I revisit this class later? Maybe the changes are gonna be like this.
```suggestion
            circle_eval_diff_cmd = Command(Command::Driver::circle-eval-diff, args, log_file=f)
            circle_eval_diff_cmd.run()
```

Reply:
Looks good to me. Hide details into Command class is a good idea.

Reply:
> Can I revisit this class later?

Of course :)

>  seems that they even don't need to be called to me because args already has the contents for the command.

FYI, `tmp_fake_quant_model` is not in args.